src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -411,6 +431,20 @@ describe('processBulkUpload', () => {
       createdAt: expect.any(Date) as Date,
     });
 
+    const [
+      proposalSubmitterEmailFormField,
+      organizationNameFormField,
+    ] = await getApplicationFormFieldsByBaseFieldIds(
+      1,
+      [1, 2],
```

Comment:
Just to call out: these ID's are based on the *test's* insertion, which is deterministic.